dockstore-cli-integration-testing/src/test/resources/config_for_singularity

```diff
@@ -0,0 +1,6 @@
+token: 238e064a9e43328b75023263d9439d880fac5829223fe7a918be69b48484f34d
```

Comment:
where is this token from? 
It looks like a dockstore token, so probably not concerning but let's double-check and/or comment

Reply:
copied from the already existing config_file2.txt

Reply:
Ok, I don't think `config_file2.txt` is normally checked-in i.e. it is part of the confidential bundle. 
That said, this token isn't live in any way (i.e. it doesn't seem to exist in the production DB) so I think this is fine.

Reply:
Should the token be regenerated and this change to a conf bundle test?